bigquery_schema_generator/generate_schema.py

```diff
@@ -486,25 +486,25 @@ def flatten_schema(self, schema_map):
             sorted_schema=self.sorted_schema,
             infer_mode=self.infer_mode)
 
-    def run(self):
+    def run(self, input_file=sys.stdin, output_file=sys.stdout):
         """Read the data records from the STDIN and print out the BigQuery
         schema on the STDOUT. The error logs are printed on the STDERR.
         """
         # TODO: BigQuery is case-insensitive with regards to the 'name' of the
         # field. Verify that the 'name' is unique regardless of the case.
 
-        schema_map, error_logs = self.deduce_schema(sys.stdin)
+        schema_map, error_logs = self.deduce_schema(input_file)
 
         for error in error_logs:
             logging.info("Problem on line %s: %s", error['line'], error['msg'])
 
         if self.debugging_map:
             json.dump(schema_map, sys.stdout, indent=2)
-            print()
+            # print()
         else:
             schema = self.flatten_schema(schema_map)
-            json.dump(schema, sys.stdout, indent=2)
-            print()
+            json.dump(schema, output_file, indent=2)
+            # print()
```

Comment:
Same here. Please remove this comment and retain the original `print()`. Without it, the JSON output does not contain a newline character on the last line.

Reply:
Same here, change to
```python
print(file=output_file)
```